data.py

```diff
@@ -20,14 +22,28 @@ def collate_fn(samples):
         strokes_mask[len(strokes[i]):,i] = 0
 
     return all_texts, texts_mask, all_strokes, strokes_mask
-    
+
+def infinite_dl(dl):
+    while True:
+        for batch in iter(dl):
+            yield batch
+
+def align_stroke(stroke):
```

Comment:
This is an interesting solution to the task. I saw the samples and it seems like it works well. However, one issue with this way of doing things is that we're modifying the empiric relation between x and y coordinates.

How could we do this in a way that would preserve the relation between x and y untouched?

Reply:
Good point. I guess we could use the avg to calculate an angle of rotation, and then update each point.